<ui version="4.0" >
 <class>ConfigViewDglClass</class>
 <widget class="QDialog" name="ConfigViewDglClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>qVortaro - Ansicht speichern</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>4</number>
   </property>
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame" >
     <property name="minimumSize" >
      <size>
       <width>16</width>
       <height>75</height>
      </size>
     </property>
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>0</number>
      </property>
      <property name="leftMargin" >
       <number>0</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>0</number>
      </property>
      <property name="bottomMargin" >
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_7" >
        <property name="minimumSize" >
         <size>
          <width>0</width>
          <height>75</height>
         </size>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="pixmap" >
         <pixmap resource="../qvortaro.qrc" >:/dlgTopVortaro</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="caption" >
        <property name="styleSheet" >
         <string>font-weight:bold;
color: #333;</string>
        </property>
        <property name="text" >
         <string>Ansicht speichern  </string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <property name="spacing" >
      <number>4</number>
     </property>
     <property name="leftMargin" >
      <number>4</number>
     </property>
     <property name="rightMargin" >
      <number>4</number>
     </property>
     <property name="bottomMargin" >
      <number>4</number>
     </property>
     <item>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>4</number>
       </property>
       <item>
        <widget class="QLabel" name="label_name" >
         <property name="text" >
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="name" />
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="cbPosSize" >
       <property name="text" >
        <string>Fenstergröße und -position speichern</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="zeigeLanguage2" >
       <property name="text" >
        <string>Wörterbuch 2 anzeigen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gbLexika" >
       <property name="title" >
        <string>Wörterbuchauswahl speichern</string>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>4</number>
        </property>
        <property name="leftMargin" >
         <number>4</number>
        </property>
        <property name="topMargin" >
         <number>4</number>
        </property>
        <property name="rightMargin" >
         <number>4</number>
        </property>
        <property name="bottomMargin" >
         <number>4</number>
        </property>
        <item>
         <layout class="QHBoxLayout" >
          <property name="spacing" >
           <number>4</number>
          </property>
          <item>
           <widget class="QLabel" name="label_5" >
            <property name="text" >
             <string>Wörterbuch 1:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="language1" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="spacing" >
           <number>4</number>
          </property>
          <item>
           <widget class="QLabel" name="label_6" >
            <property name="text" >
             <string>Wörterbuch 2:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="language2" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <layout class="QGridLayout" >
         <property name="horizontalSpacing" >
          <number>4</number>
         </property>
         <property name="verticalSpacing" >
          <number>4</number>
         </property>
         <item row="1" column="0" >
          <widget class="QLabel" name="label" >
           <property name="text" >
            <string>Wörterbücher:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QComboBox" name="cLexika" >
           <item>
            <property name="text" >
             <string>nebeneinander</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>untereinander</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="label_2" >
           <property name="text" >
            <string>Wörter und Übersetzung:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="QComboBox" name="cLexikaObjekte" >
           <item>
            <property name="text" >
             <string>nebeneinander</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>untereinander</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="0" >
          <widget class="QLabel" name="label_3" >
           <property name="text" >
            <string>Reihenfolge:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" >
          <widget class="QComboBox" name="cLB" >
           <item>
            <property name="text" >
             <string>Wörter - Übersetzung</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Übersetzung - Wörter</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="0" >
          <widget class="QLabel" name="label_8" >
           <property name="text" >
            <string>Überschrift:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QComboBox" name="cbCaption" >
           <item>
            <property name="text" >
             <string>keine</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>kurz</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>lang</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>145</width>
           <height>82</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title" >
        <string>Vorschau</string>
       </property>
       <layout class="QHBoxLayout" >
        <property name="leftMargin" >
         <number>4</number>
        </property>
        <property name="topMargin" >
         <number>4</number>
        </property>
        <property name="rightMargin" >
         <number>4</number>
        </property>
        <property name="bottomMargin" >
         <number>4</number>
        </property>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>16</width>
            <height>77</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" >
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>10</width>
              <height>10</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QGridLayout" >
            <property name="horizontalSpacing" >
             <number>4</number>
            </property>
            <property name="verticalSpacing" >
             <number>4</number>
            </property>
            <item row="0" column="0" >
             <widget class="QLabel" name="l1" >
              <property name="text" >
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <widget class="QLabel" name="l2" >
              <property name="text" >
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="2" >
             <widget class="QLabel" name="l3" >
              <property name="text" >
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>10</width>
              <height>10</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>16</width>
            <height>77</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons" >
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../qvortaro.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigViewDglClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigViewDglClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
